doc/eip.md

```diff
@@ -12,28 +12,58 @@ requires: EIP-155, EIP-1014, EIP-7702
-   - If the first transaction is not a successful CREATE2 factory deployment, than it is no longer possible to have a CREATE2 factory at the common address; this can happen by human error for example.
-3. Have popular CREATE2 deployment factories deployed on new chains by default. This is, for example, what OP Stack does as part of their [preinstalls](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Preinstalls.sol) including the CREATE2 factory contracts mentioned above. This ensures that the CREATE2 factory contracts have known address and code. The downsides to this approach are:
+   This requires carefully guarding a secret key; if it is exposed or lost, deployments are no longer guaranteed on new chains.
+   - If the first transaction is not a successful CREATE2 factory deployment, then it is no longer possible to have a CREATE2 factory at the common address; this can happen by human error, for example.
+
+#### 3. CREATE2 deployment factories
+
+Have popular CREATE2 deployment factories deployed on new chains by default. This is, for example, what OP Stack does as part of their [preinstalls](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Preinstalls.sol), including the CREATE2 factory contracts mentioned above. This ensures that the CREATE2 factory contracts have known addresses and codes.
+
+**Downsides**:
+
    - It is not standardized nor adopted by all chains.
-   - It is permissioned as a chain can choose to not include a specific CREATE2 factory contract preinstalled.
+   - It is permissioned as a chain can choose not to include a specific CREATE2 factory contract preinstalled.
    - Attempts to standardize this with [RIP-7740](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7740.md) have not been successful.
 
-This ERC proposes a permissionless alternative fourth mechanisms to the existing ones described above with none of their downsides. Additionally, it standardizes a set of deployment parameters for a **unique** CREATE2 factory deployment. This ensures a common CREATE2 factory for the community, instead of having multiple competing copies at different addresses with slightly different code. This single CREATE2 factory copy can be used to bootstrap additional deterministic deployment infrastructure (such as the aforementioned comprehensive CreateX universal contract deployer).
+This ERC proposes a permissionless alternative fourth mechanism to the existing ones described above with none of their downsides. Additionally, it standardizes a set of deployment parameters for a **unique** CREATE2 factory deployment. This ensures a common CREATE2 factory for the community instead of multiple competing copies with slightly different codes at different addresses. This single CREATE2 factory copy can bootstrap additional deterministic deployment infrastructure (such as the comprehensive CreateX universal contract deployer).
```

Comment:
I feel like this sentence and the section below have some uneccessary duplication. Also not a fan that this is technically in the '3. CREATE2 deployment factories' section despite not belonging there. Maybe it can be merged with the section below?